Gemfile

```diff
@@ -25,5 +25,6 @@ group :development, :test do
 end
 
 group :deployment do
+  gem 'capistrano-bundler'
```

Comment:
Sounds about right. We're clearly using capistrano directly, so we should have a direct dependency..